### PR TITLE
Add CMake build to GitHub CI

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

**Comment by matthiasdiener** [comment]
This is ready for review and should not have any impact on 6.10. Although the cmake test doesn't pass currently, this should be merged so that we can double-check fixes to the cmake build.

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: build
+      run: ./buildcmake LIBS netlrts-linux-x86_64 -g -j4 --with-production
+    - name: test
+      run: make -C netlrts-linux-x86_64/tests test OPTS="-g" TESTOPTS="++local +isomalloc_sync"
```
Comment (evan-charmworks) [inline .github/workflows/cmake.yml]: Should this be `make -C netlrts-linux-x86_64/tests all OPTS="-g" -j4 && make -C netlrts-linux-x86_64/tests test OPTS="-g" TESTOPTS="++local +isomalloc_sync"`?

Comment (matthiasdiener) [reply .github/workflows/cmake.yml]: Fixed. I removed the second 'OPTS'.

**Comment by matthiasdiener** [comment]
Needs another review.

**Pull request merged**